package.json

```diff
@@ -56,40 +92,49 @@
       "last 1 safari version"
     ]
   },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "dependencies": {
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#PP-357/dapp-update",
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#feature/PP-278/PP-329/upgrade_contracts",
+    "@rsksmart/rlogin": "^1.4.1-beta.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.10",
+    "@types/react": "^17.0.39",
```

Comment:
Needs to be downgraded so the `react-materialize` modals can work without complaining about script.